compio-io/src/write/dyn.rs

```diff
@@ -0,0 +1,40 @@
+use compio_buf::{BufResult, IoBuf, IoVectoredBuf};
+
+use crate::{util::BoxFuture, AsyncWrite};
```

Comment:
Where is BoxFuture?

And, should we use LocalBoxFuture?

Reply:
In out context there's no global. Everything except the dispatcher should be local so this is fine.